prov/verbs/src/verbs_eq.c

```diff
@@ -164,6 +165,9 @@ fi_ibv_eq_cm_process_event(struct fi_ibv_eq *eq, struct rdma_cm_event *cma_event
 	case RDMA_CM_EVENT_ESTABLISHED:
 		*event = FI_CONNECTED;
 		entry->info = NULL;
+		ret = fi_ibv_modify_qp(cma_event->id->qp);
```

Comment:
Even the unexpected queue requires posting receives to the QP.   I think the change can apply there as well.  Not sure where *there* is though.